client/globals.d.ts

```diff
@@ -17,6 +17,7 @@ declare global {
 			paymentTimeline: boolean;
 			isDisputeIssuerEvidenceEnabled: boolean;
 			isPaymentOverviewWidgetEnabled?: boolean;
+			isMerchantFeedbackPromptEnabled: boolean;
```

Comment:
Is this the same as the [server side eligibility flag](https://github.com/Automattic/transact-platform-server/pull/7246) (i.e. is this a placeholder)?

Reply:
This is a client-only dev flag, needed to allow us to push in-development code to trunk. I've renamed to clarify in in 6d0e870.